packages/fuel-indexer-api-server/src/uses.rs

```diff
@@ -86,7 +86,7 @@ pub(crate) async fn get_fuel_status(config: &IndexerConfig) -> ServiceStatus {
                 .await
                 .unwrap_or_default();
 
-            let fuel_node_health: FuelNodeHealthResponse =
+            let fuel_node_health: FuelClientHealthResponse =
```

Comment:
I see that we're referring to `FuelClient*`, which is clearer; can we also rename `fuel_node_health` to `fuel_client_health` to match?

Reply:
@deekerno  That's the thing. If we rename _everything_ to `FuelClient` the footprint is obviously much larger, and includes a few breaking changes. I originally actually had this (changing _everything_) but felt that was a little too large of a footprint for this cleanup - and that should be in its own separate breaking PR.

@lostman If we're all aligned to change these namings to `FuelClient*` let's do that separately